<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketProductoWidget</class>
 <widget class="QWidget" name="TicketProductoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblNombre">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPrecio">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>15</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCantidad">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>15</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
